src/components/interaction-designer/block-types/Core_CaseBlock.vue

```diff
@@ -47,7 +47,7 @@ const blockVuexNamespace = namespace(`flow/${BLOCK_TYPE}`)
 const builderVuexNamespace = namespace('builder')
 
 @Component({})
-class Core_CaseBlock extends mixins(Lang) {
+export class Core_CaseBlock extends mixins(Lang) {
```

Comment:
these are not really important, I added them to investigate on some export issue. I think I will leave them as it is.